wonnx/examples/mnist.rs

```diff
@@ -26,10 +25,7 @@ async fn execute_gpu() -> Option<HashMap<String, Vec<f32>>> {
     let mut input_data = HashMap::new();
 
     let image = load_image();
-    input_data.insert(
-        "Input3".to_string(),
-        InputTensor::F32(image.as_slice().unwrap()),
-    );
+    input_data.insert("Input3".to_string(), image.as_slice().unwrap().into());
```

Comment:
I think we have to update the README accordingly. We can maybe reference this file in order to avoid having to manually update the README

Reply:
Agreed. (And now I am wondering if Rust will run doctests (i.e. check if the code in documentation actually runs) if you use the include_str trick...)